tests/agent/actions/test_draft_datasource_config.py

```diff
@@ -0,0 +1,169 @@
+import os
+from unittest.mock import MagicMock
+from uuid import UUID
+
+import pytest
+
+from great_expectations.agent.actions.test_draft_datasource_config import (
+    DraftDatasourceConfigAction,
+)
+from great_expectations.agent.agent import GxAgentEnvVars
+from great_expectations.agent.models import DraftDatasourceConfig
+from great_expectations.data_context import CloudDataContext
+from great_expectations.datasource.fluent import SQLDatasource
+from great_expectations.datasource.fluent.interfaces import TestConnectionError
+
+
+@pytest.fixture(scope="function")
+def context():
+    return MagicMock(autospec=CloudDataContext)
+
+
+@pytest.fixture
+def org_id():
+    return "81f4e105-e37d-4168-85a0-2526943f9956"
+
+
+@pytest.fixture
+def token():
+    return "MTg0NDkyYmYtNTBiOS00ZDc1LTk3MmMtYjQ0M2NhZDA2NjJk"
+
+
+@pytest.fixture(autouse=True)
+def set_required_env_vars(monkeypatch, org_id, token):
+    env_vars = {
+        "GX_CLOUD_ORGANIZATION_ID": org_id,
+        "GX_CLOUD_ACCESS_TOKEN": token,
+    }
+    monkeypatch.setattr(os, "environ", env_vars)
```

Comment:
Please use the `monkeypatch.setenv` for this.
https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html#monkeypatching-environment-variables

Reply:
thanks! [90923bc](https://github.com/great-expectations/great_expectations/pull/8410/commits/90923bc765c227a1e940d35d7ad9c8052b1e77dd)